src/cli/exceptions.py

```diff
@@ -1,2 +1,49 @@
 class MalformedJsonFile(Exception):
     """ Raised when trying to read a file containing malformed json """
+
+
+class ApplicationAlreadyExists(Exception):
+    """ Raised when application name is not unique and already exists in .qne/application.json"""
+
+    # TODO: Can't they use application delete command instead of giving the path location?
```

Comment:
Temporary answer: Depends on when the user wants to restart the development of the same app or not.
The user could use application list to get the path of the application.
Maybe, later we can decide to delete the hint where the path is (let's find out by using the cli).